build.gradle

```diff
@@ -12,5 +12,8 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        maven {
+            url "https://maven.google.com"
+        }
```

Comment:
Its already available, see https://github.com/freeotp/freeotp-android/commit/5cf403c91d44965f3fcb8e8a74f3d7783743ae13

Reply:
The `Google()` gradle function is only defined in gradle 4+ IIRC, I prefer the explicit url as it works for newer and older IDEs.

It's not really a big deal though.